locale/ca_ES/locale.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * locale/ca_US/locale.xml
+  *
+  * Copyright (c) 2015 Simon Fraser University Library
+  * Copyright (c) 2003-2015 John Willinsky
+  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+  *
+  * Credits: http://pkp.sfu.ca/wiki/index.php?title=OMP:_Catalan_%28ca_ES%29
+  *
+  * Localization strings
+  -->
+ 
+<locale name="ca_ES" full_name="Català">
+	<message key="plugins.generic.staticPages.displayName">Mòdul de pàgines estàtiques</message>
+	<message key="plugins.generic.staticPages.description">Aquest mòdul permet gestionar el contingut estàtic.</message>
+	<message key="plugins.generic.staticPages.pageTitle">Títol</message>
+	<message key="plugins.generic.staticPages.content">Contingut</message>
+
+	<message key="plugins.generic.staticPages.path">Ruta</message>
+	<message key="plugins.generic.staticPages.editAddContent">Edita/Afegeix contingut</message>
+
+	<message key="plugins.generic.staticPages.addStaticPage">Afegeix una pàgina estàtica</message>
+	<message key="plugins.generic.staticPages.addNewPage">Afegeix una nova pàgina</message>
+	<message key="plugins.generic.staticPages.editStaticPage">Edita una pàgina estàtica</message>	
+
+	<message key="plugins.generic.staticPages.editInstructions">Utilitzeu el formulari que segueix a continuació per a afegir o editar contingut de pàgina.</message>
+	<message key="plugins.generic.staticPages.duplicatePath">Aquesta ruta ja pertany a una altra pàgina estàtica.</message>
+	
+	<message key="plugins.generic.staticPages.pageSaved">S&apos;ha desat la pàgina correctament.</message>
```

Comment:
You are right. BTW... same with &quot;
Better chars and UTF or we can use entities?

I'm asking because I'm reviewing all files, one by one (to fix headers, remove en_US, etc) and is a good moment to intercept this.